install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -21,17 +21,21 @@ data:
         - {{ "{{ .MeshConfig.ProxyListenPort }}" }}
         - "-u"
         - 1337
-        {{ if .Values.includeIPRanges -}}
         - "-i"
+        {{ if .Values.includeIPRanges -}}
         - {{ .Values.includeIPRanges | quote }}
+        {{ else -}}
+        - "*"
```

Comment:
Even quoted I'm a bit scared of *. 
Since we plan to clean this up anyways - it's ok for now.

Reply:
I've confirmed that it comes across as just `*` in the script.